scripts/gen_vimdoc.py

```diff
@@ -54,6 +59,18 @@
     print("requires Python 3.5+")
     sys.exit(1)
 
+if not shutil.which('doxygen'):
+    print("Missing Requirement: doxygen")
+    sys.exit(1)
+
+if not any(
+    shutil.which(x) for x in {'lua', 'luajit', 'texlua', './.deps/usr/bin/luajit'}
```

Comment:
Should probably prefer any `.deps` binaries, in case there are deps related modules that we need.  Also check for `lua` in `.deps`, since we can build with that instead of luajit.

Reply:
Yeah, it actually does prefer the `.deps`. Didn't know it could be built with just `lua` in deps, so I'll add that here and to the script that actually chooses which lua to use.

Maybe I will refactor to run that script and make sure it actually returns something instead of keeping them in two places.